filebeat/_meta/beat.full.yml

```diff
@@ -66,30 +66,30 @@ filebeat.prospectors:
   # of under the "fields" sub-dictionary. In case of name conflicts with the
   # fields added by Filebeat itself, the custom fields overwrite the default
   # fields.
-  #fields_under_root: false
+  fields_under_root: false
 
   # Ignore files which were modified more then the defined timespan in the past.
   # ignore_older is disabled by default, so no files are ignored by setting it to 0.
   # Time strings like 2h (2 hours), 5m (5 minutes) can be used.
-  #ignore_older: 0
+  ignore_older: 0
 
   # Type to be published in the 'type' field. For Elasticsearch output,
   # the type defines the document type these entries should be stored
   # in. Default: log
-  #document_type: log
+  document_type: log
 
   # How often the prospector checks for new files in the paths that are specified
   # for harvesting. Specify 1s to scan the directory as frequently as possible
   # without causing Filebeat to scan too frequently. Default: 10s.
-  #scan_frequency: 10s
+  scan_frequency: 10s
 
   # Defines the buffer size every harvester uses when fetching the file
-  #harvester_buffer_size: 16384
+  harvester_buffer_size: 16384
 
   # Maximum number of bytes a single log event can have
   # All bytes after max_bytes are discarded and not sent. The default is 10MB.
   # This is especially useful for multiline log messages which can get large.
-  #max_bytes: 10485760
+  max_bytes: 10485760
 
   ### JSON configuration
```

Comment:
@elastic/beats Below we have json and multiline which are completely commented out. Brings up the question if it is worth to have enabled / disable also for these?